realm/ownable.gno

```diff
@@ -0,0 +1,23 @@
+package chess
+
+import "std"
+
+func TransferOwnership(newOwner std.Address) string {
+	if !isOwner() {
+		panic("not owner")
+	}
+	owner = newOwner
+	return "new owner is " + string(owner)
+}
+
+func isOwner() bool {
+	caller := std.GetOrigCaller()
+	if caller != GetOwner() {
+		return false
+	}
+	return true
+}
+
+func GetOwner() std.Address {
```

Comment:
Why is this exported?

Reply:
So that it can be called by gnokey, as to check who is the owner of realm